src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -79,13 +79,17 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
                         <CustomVideoPlayerButton order={1.1}
                             accelerators={["ArrowLeft", "a", "A"]}
                             tooltip={strings.editorPage.videoPlayer.previousExpectedFrame.tooltip}
-                            onClick={this.movePreviousExpectedFrame}>
+                            onClick={this.movePreviousExpectedFrame}
+                            icon={"fa-caret-left fa-lg"}
```

Comment:
We need this because the underlying `KeyboardBinding` accepts an icon that can be displayed in the help menu. Not required, but it provides the user with not only the help for the key binding, but the icon itself as well